.gitlab-ci.yml

```diff
@@ -214,6 +214,30 @@ check_changelog:
         exit 0
       fi
 
+migrate:
+  stage: test
+  needs:
+    - job: install
+  cache:
+    - <<: *ruby_cache
+  variables:
+    DOCKER_DB_HOST: db-postgres
+    POSTGRES_DB: identity_idp_test
+    POSTGRES_USER: postgres_user
+    POSTGRES_PASSWORD: postgres_password
+    POSTGRES_HOST_AUTH_METHOD: trust
+    RAILS_ENV: test
+  services:
+    - name: postgres:13.9
+      alias: db-postgres
+      command: ['--fsync=false', '--synchronous_commit=false', '--full_page_writes=false']
+  script:
+    - *bundle_install
+    - bundle exec rake db:create db:migrate --trace
+    - git diff db/
+    - '(! git diff --name-only | grep db/schema.rb) || (echo "Error: db/schema.rb does not match after running migrations"; exit 1)'
+    - '(! git diff --name-only | grep db/worker_jobs_schema.rb) || (echo "Error: db/worker_jobs_schema.rb does not match after running migrations"; exit 1)'
```

Comment:
should we move these into a Makefile command so it's easy to test/verify on development boxes?

Reply:
I like Zach's suggestion if it's an easy change, but otherwise +1 to this.

Reply:
I was trying to think how to approach it with the database creation and running migrations. Would we include that in the Makefile or just the diff commands?

Reply:
just the diff commands probably?

Reply:
Makes sense, added in771da4c0cc